### ISSUE TITLE
FEATURE REQUEST - Kodi Media Path - unRAID

### ISSUE THREAD

**Issue by hernandito** [matthuisman/docker-kodi-headless#11]
Thank you for keeping this alive! 

**EDIT:** Never mind: adding the below to the unRAID template did the trick!!!

![image](https://user-images.githubusercontent.com/787317/154959770-7fb8d9b5-1d7a-4d04-8a61-a2e57fa007a5.png)

 PLEASE IGNORE - See my edit above
================================================
_I use this Docker in my unRAID server to scan for library updates. In the readme you outline a method for faster scans when the the docker and media storage are on the machine.

can you please modify both the Dockerfile and the unRAID template so that my /mnt/user/Media (can be user-modified) folder could be mapped into a folder inside the dockers /config/media folder.

Basically build in this function into the template
`eg. --mount type=bind,source=/sharedfolders/pool,target=/media `_

i don’t think adding a new path in the template screen works. 

Thank you again!
 
 h.
